.github/workflows/scanner-update-nvd.yaml

```diff
@@ -0,0 +1,52 @@
+name: Scanner NVD update
+
+on:
+  push:
+    branches:
+      - yli3/**
+  schedule:
+    - cron: '0 */4 * * *'  # this runs every 4 hours
+  workflow_dispatch:
+
+jobs:
+  fetch-cves:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up Python
```

Comment:
This script might be temporary since Victor's has a PR for Claicore jsonblob and offline import. So we might be able to bundle the CVSS data with the vuln data in the future and inject to Claircore via offline import. @RTann 
https://github.com/quay/claircore/pull/1166